frontend/src/routes/Infrastructure/Clusters/ClusterPools/CreateClusterPool/CreateIntrastructureClusterpool.tsx

```diff
@@ -0,0 +1,116 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { CatalogCardItemType, CatalogColor, ICatalogCard, ItemView, PageHeader } from '@stolostron/react-data-view'
+import { Fragment, useCallback, useMemo } from 'react'
+import { useHistory } from 'react-router-dom'
+import { useRecoilState } from 'recoil'
+import { secretsState } from '../../../../../atoms'
+import { useTranslation } from '../../../../../lib/acm-i18next'
+import { NavigationPath } from '../../../../../NavigationPath'
+import { AcmIcon, AcmIconVariant, Provider } from '../../../../../ui-components'
+
+export function CreateInfrastructureClusterpool() {
+    const [t] = useTranslation()
+    const history = useHistory()
+    const [secrets] = useRecoilState(secretsState)
+    const credentials = useMemo(
+        () =>
+            secrets.filter(
+                (secret) => secret?.metadata?.labels?.['cluster.open-cluster-management.io/credentials'] !== undefined
+            ),
+        [secrets]
+    )
+
+    const getCredentialLabels = useCallback(
+        (provider: Provider) => {
+            return credentials.filter(
+                (secret) => secret?.metadata?.labels?.['cluster.open-cluster-management.io/type'] === provider
+            ).length > 0
+                ? [{ label: t('Saved credentials'), color: CatalogColor.green }]
+                : undefined
+        },
+        [credentials, t]
+    )
+
+    const cards = useMemo(() => {
+        const newCards: ICatalogCard[] = [
+            {
+                id: 'aws',
+                icon: <AcmIcon icon={AcmIconVariant.aws} />,
+                title: t('Amazon Web Services'),
+                items: [
+                    {
+                        type: CatalogCardItemType.Description,
+                        description: t('A Red Hat OpenShift cluster that is running in your AWS subscription.'),
```

Comment:
The description needs to be changed since we're creating a clusterpool now. I would suggest:
`A Red Hat OpenShift clusterpool that is running in your AWS subscription.`

But maybe check with the design or doc team to see what they think.